crates/swc_atoms/words.txt

```diff
@@ -870,8 +870,8 @@ XPathExpression
 XPathResult
 XSLTProcessor
 __proto__
-_define_property
 _defineProperty
```

Comment:
Is it safe to remove `_defineProperty` now?

Reply:
Technically it's a breaking change because `js_word!("_defineProperty")` will not compile anymore, but I think it's fine.